tasks/shared_libraries/requirements.txt

```diff
@@ -0,0 +1,15 @@
+## Library used for documentation
+pydoc-markdown>=4.0.0,<5.0.0
+## Additional validation libraries
+
+## Standard unit test libraries
+pytest==6.1.2
+coverage==5.3
+## Libraries needed for unit tests
+moto[sqs]==2.0.6
+psycopg2-binary==2.8.6
+## Libraries needed by application
```

Comment:
Might be a silly question, but which application?

Reply:
Added more comments and broke items up to better detail what requirements belong to what.